Sources/TootSDK/HTTP/HashtagTimelineQuery.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  HashtagTimelineQuery.swift
+//  
+//
+//  Created by Dale Price on 3/7/23.
+//
+
+import Foundation
+
+/// Specifies the parameters for a hashtag timeline request
+public struct HashtagTimelineQuery: Codable, Sendable {
```

Comment:
Could you include a public initializer with all fields and a perhaps a second initializer taking a single string tag:

```swift
HashtagTimelineQuery(tag: "", anyTags:...)
HashtagTimelineQuery(tag: "")
```

Reply:
Whoops, forgot the default initializer is `internal`. Fixed in aeb4a0c